drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/TestToolsPresenter.java

```diff
@@ -370,6 +370,12 @@ public void onModifyColumn() {
         }
     }
 
+    @Override
```

Comment:
@yesamer 
Please adapt it to the above comments

Reply:
Done